terraform/services/zscaler-security-groups/main.tf

```
@@ -0,0 +1,23 @@
+resource "aws_security_group" "zscaler_traffic_allow_public" {
```

"traffic_allow" is unnecessary as that's what a security group generally does

```suggestion
resource "aws_security_group" "zscaler_public" {
```